### ISSUE TITLE
Feedparser is not working.

### ISSUE THREAD

**Issue by PeterLeen** [custom-components/feedparser#53]
I installed feedparser and to try it out I used the example configuration.

When I visit https://www.sciencedaily.com/rss/matter_energy/engineering.xml I see the feed.

But in Home Assistant I see nothing.

**Comment by Luki72** [comment]
Similar problem here:

I use feedparser to fetch some RSS feeds, eg: https://hardverapro.hu/hirfolyam/aprok/mobil/tablet_ebook/android_tablet/10_felett/rss.xml

Associated configuration:

```
- platform: feedparser
  name: HARDVERAPRÓ - Tablet 10 felett
  feed_url: "https://hardverapro.hu/hirfolyam/aprok/mobil/tablet_ebook/android_tablet/10_felett/rss.xml"
  date_format: '%b %w, %a %I:%M'
  show_topn: 10
  inclusions:
    - title
    - link
    - pubDate
```

The appropriate entity, **sensor.hardverapro_tablet_10_felett exists**, it's status is **unknown** and it's Etries attribute contains the right data except that **pubDate** field is *missing*. The first entry looks like this:

```
- title: Samsung Tab S4 LTE, 4/64GB, HIBÁTLAN, KARCMENTES állapotú eladó!
link: >-
https://hardverapro.hu/apro/samsung_tab_s4_lte_4_64gb_hibatlan_karcmentes_alla/hsz_1-50.html

```
I run an up-to-date (2012.12.0 @ the moment) supervised installation on Debian.

**Comment by iantrich** [comment]
should be fixed in 0.1.8
